administrator/components/com_admin/sql/updates/postgresql/6.0.0-2025-08-26.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Add position column to workflow stages table
+--
+
+ALTER TABLE "#__workflow_stages" ADD COLUMN "position" text DEFAULT NULL;
```

Comment:
```suggestion
ALTER TABLE "#__workflow_stages" ADD COLUMN "position" text NULL /** CAN FAIL **/;
```
See description of PR #41184 for explanation of /** CAN FAIL **/.

The default value is removed here as it it not necessary. The NULL remains to allows NULL values for the column.

Reply:
In addition, this update SQL script has to be renamed from `6.0.0-2025-08-26.sql` to `6.1.0-2025-08-26.sql`.